VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -52,7 +51,7 @@ public CoursesAsyncLoader(Context context) {
                 environment.getNotificationDelegate().checkCourseEnrollment(enrolledCoursesResponse);
             }
 
-        } catch (HttpException exception) {
+        } catch (Exception exception) {
```

Comment:
Is there any way to make this catch less global? Maybe we can just catch all the exception types applicable?